### ISSUE TITLE
Getting 403 suddenly in app22 sandbox

### ISSUE THREAD

**Issue by ThomasNYoung** [GEHealthcare/apiserver.gehealthcare.com#322]
I have been doing testing in my local environment, and through Insomnia(like Postman). 
It had been working fine. I am able to get an access token using:
https://ap22sandbox.fhirapi.athenahealth.com/demoAPIServer/oauth2/token

https://ap22sandbox.fhirapi.athenahealth.com/demoAPIServer/fhir/Location/
Now any other end point I try to access with the returned access token gives me a 403:
"issue": [
		{
			"severity": "error",
			"code": "processing",
			"diagnostics": "Not authorized"
		}
	]

Has something changed? Again I had this working just fine before, and haven't made any changes on my end that would cause a 403.

**Comment by deepaktiwari29** [comment]
Hello @ThomasNYoung ,
Thank you for reaching out to us. Could you please re-verify this once more on your end, because I am able to the expected response on my end.

**Comment by ThomasNYoung** [comment]
Yes, the authentication started working again, but now when I try to pull appointment data, there does not seem to be any appointments associated with any of the providers or locations in the sandbox. Several weeks ago there were quite a few test appointments associated with both the providers and locations. 

Any advice would be helpful.

**Comment by deepaktiwari29** [comment]
Hey @ThomasNYoung,
You might not be able to get any appointment-related data because we removed all test data around December in order to apply our migration policy. However, you should still be able to use our APIs to create new Appointments.
If you still need any information or if there is anything else we can do to assist you, do let us know.

**Comment by deepaktiwari29** [comment]
@ThomasNYoung Hoping we have answered your question. Marking this issue as closed, if you have any further questions, please open a new ticket.
